src/components/DataContainer/FlourishDataContainer.js

```diff
@@ -53,7 +53,7 @@ function DataContainer({ id, data, theme, countryName, url }) {
 
   const handleShare = () => {
     toCanvas().then(canvas => {
-      uploadImage(id, canvas.toDataURL('image/png')).then(success => {
+      uploadImage(id, canvas.toDataURL('image/png'), url).then(success => {
```

Comment:
You know know me, if this variable will never change why not uploadImage import config and use the url instead of passing this down.